### PR TITLE
fix: update happy version for all GHAs

### PR DESCRIPTION
**Pull request by danieljhegeman**
## Reason for Change

- #5925 

## Changes

- update happy version in all GHAs to `0.110.1`
- change branch type from `open` to `opened` for GHA trigger (bug that prevents rdev workflow for single-commit PR's?)

## Testing steps

- Either list QA steps or reasoning you feel QA is unnecessary
- Describe how you made sure to know that your changes worked. Should allow someone else to go verify your code without in depth knowledge.
- "Unit tested only", "tested in rdev by a, b, c, verifying feature worked by... ", "manually ran pipeline locally with these results: ..."

### CODE REVIEW

**Review by ebezzi — APPROVED** [review]

File: .github/workflows/rdev-update-for-pr.yml [hunk 1]
```diff
@@ -247,7 +247,7 @@ jobs:
       - name: Install happy
         uses: chanzuckerberg/github-actions/.github/actions/install-happy@install-happy-v1.4.2
         with:
-          happy_version: "0.59.0"
```
Comment (Bento007) [inline .github/workflows/rdev-update-for-pr.yml:250]: This might greak

Comment (danieljhegeman) [reply .github/workflows/rdev-update-for-pr.yml:250]: 🤞

File: .github/workflows/rdev-update-for-pr.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ on:
     branches-ignore:
       - prod
     types:
-      - open
+      - opened
```
Comment (danieljhegeman) [inline .github/workflows/rdev-update-for-pr.yml:8]: @Bento007 I think this was a bug that prevents rdevs from being created for single-commit new PR's. The rdev workflow was not triggered for me.

Comment (Bento007) [reply .github/workflows/rdev-update-for-pr.yml:8]: close and reopen will fix it

**Pull request merged**